kotlin/vector-tiled-layer-custom-style/src/main/java/com/esri/arcgisruntime/sample/vectortiledlayercustomstyle/MainActivity.kt

```diff
@@ -0,0 +1,313 @@
+package com.esri.arcgisruntime.sample.vectortiledlayercustomstyle
```

Comment:
```suggestion
package com.esri.arcgisruntime.sample.arcgisvectortiledlayercustomstyle
```

Sorry, one more name change across the sample